_use-release-management.shade

```diff
@@ -180,19 +192,23 @@
         foreach (var repo in GetAllRepos())
         {
             GitCommand(repo, "pull --tags");
+            string version = null;
 
             try
             {
                 GitCommand(repo, string.Format("describe --tags > ..\\{0}", versionFile));
             }
             catch
             {
-                Log.Warn(string.Format("{0} repo not tagged. Skipping....", repo));
-                continue;
+                version = "1.0.0-" + preReleaseTag;
```

Comment:
Workaround for some repos that don't get tagged by @danroth27 (such as Coherence, Proxy etc)